decentraland/unity-renderer
# fix: changed open sea nft endpoints

## What does this PR change?

Following this [thread](https://decentralandteam.slack.com/archives/C01QVR7TJK1/p1666622942043589), we decided to change the opensea endpoints to lambda functions.

## How to test the changes?

1. Go to the NFT gallery in Genesis Plaza. Everything should be displayed correctly
2. Go to 26,-118. There is a NFT Gallery on the second floor of TruBand. Check that is displayed correctly. There are more galleries in the neighbor parcels (DJTrax, mirrorvision)
3. If you have another NFT gallery in mind, go and check it as well :)

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
